src/Main.jsx

```diff
@@ -1,12 +1,14 @@
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import App from '@/App';
 import NotFound from '@/pages/NotFound';
+import Landing from '@/pages/Landing'
 
 function Main() {
   return (
     <BrowserRouter>
       <Routes>
         <Route path='/' element={<App />}>
+        <Route path='/landing' element={<Landing />} />
```

Comment:
<Route path='/' element={<App />}> 하위에 들어 있기 때문에 path='landing' 이렇게만 써주셔도 됩니다~
```suggestion
<Route path='landing' element={<Landing />} />
```

Reply:
앗 그러네요